venia-integration-tests/src/actions/wishlist/index.js

```diff
@@ -0,0 +1,85 @@
+import {
+    wishlistPageHeading,
+    wishlistRoot,
+    wishlistItemPrice,
+    wishlistItemAddToCartSection,
+    wishlistItemAddToCartButton,
+    wishlistItemMoreActionsButton,
+    removeProduct,
+    deleteProduct
+} from '../../fields/wishlist';
+import { wishlistRemove, removeProductMessage } from '../../fixtures/wishlist';
+
+/**
+ * Utility function to assert empty wishlist
+ * @param {String} wishlistHeaderText wishlist page header text
+ */
+export const assertWishlistHeading = wishlistHeaderText => {
```

Comment:
Please move assertions to the assertions folder.

Reply:
Dont know how I ended up here but updated all, thanks. Please re-check :)